.gitignore

```diff
@@ -451,4 +451,7 @@ secring.*
 # Generated version
 /urwid/version.py
 # Built packages
-/wheelhouse
\ No newline at end of file
+/wheelhouse
+
+### Documentation build
+/docs/source
```

Comment:
/docs/build?

Reply:
and does the build still work from the new directory? I remember there were a few places in the docs that referenced the source, maybe those are broken by moving the files.

Reply:
```
~/PycharmProjects/urwid (docs_build?) $ tox -e docs
.pkg: _optional_hooks> python /home/alekseis/.virtualenv/urwid/lib/python3.11/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: get_requires_for_build_editable> python /home/alekseis/.virtualenv/urwid/lib/python3.11/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: build_editable> python /home/alekseis/.virtualenv/urwid/lib/python3.11/site-packages/pyproject_api/_backend.py True setuptools.build_meta
docs: install_package> python -I -m pip install --force-reinstall --no-deps /home/alekseis/PycharmProjects/urwid/.tox/.tmp/package/98/urwid-2.2.2.dev3+g4cde7ee-0.editable-cp311-cp311-linux_x86_64.whl
docs: commands[0]> sphinx-build docs/source docs/build
Running Sphinx v7.2.6
making output directory... done
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 29 source files that are out of date
updating environment: [new config] 29 added, 0 changed, 0 removed
reading sources... [100%] tutorial/index
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
copying assets... copying static files... done
copying extra files... done
done
/home/alekseis/PycharmProjects/urwid/urwid/widget/widget_decoration.py:docstring of urwid.widget.widget_decoration.WidgetDecoration.selectable:10: WARNING: more than one target found for cross-reference 'keypress': urwid.Widget.keypress, urwid.Edit.keypress, urwid.IntEdit.keypress, urwid.Button.keypress, urwid.CheckBox.keypress, urwid.TreeWidget.keypress, urwid.SelectableIcon.keypress, urwid.Padding.keypress, urwid.Filler.keypress, urwid.Frame.keypress, urwid.ListBox.keypress, urwid.TreeListBox.keypress, urwid.Columns.keypress, urwid.Pile.keypress, urwid.GridFlow.keypress, urwid.Overlay.keypress
writing output... [100%] tutorial/index
/home/alekseis/PycharmProjects/urwid/urwid/widget/widget_decoration.py:docstring of urwid.widget.widget_decoration.WidgetDisable.selectable:10: WARNING: more than one target found for cross-reference 'keypress': urwid.Widget.keypress, urwid.Edit.keypress, urwid.IntEdit.keypress, urwid.Button.keypress, urwid.CheckBox.keypress, urwid.TreeWidget.keypress, urwid.SelectableIcon.keypress, urwid.Padding.keypress, urwid.Filler.keypress, urwid.Frame.keypress, urwid.ListBox.keypress, urwid.TreeListBox.keypress, urwid.Columns.keypress, urwid.Pile.keypress, urwid.GridFlow.keypress, urwid.Overlay.keypress
generating indices... genindex done
writing additional pages... index search done
copying images... [100%] tutorial/adventure4.png
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 2 warnings.

The HTML pages are in docs/build.
.pkg: _exit> python /home/alekseis/.virtualenv/urwid/lib/python3.11/site-packages/pyproject_api/_backend.py True setuptools.build_meta
  docs: OK (5.38=setup[0.86]+cmd[4.52] seconds)
  congratulations :) (5.44 seconds)
```

Reply:
@wardi added documentation build pipeline, artifact with documentation will be available for 90 days after build.
Publishing for documentation should be done separately.